docs-app/app/components/ui/form/information.hbs

```diff
@@ -1,4 +1,4 @@
-{{#if (or @title @instructions)}}
+{{#if (strict-or @title @instructions)}}
```

Comment:
Ideally, the `{{strict-or}}` helper might allow the input conditionals to be `boolean` or `undefined` (and no other type is allowed).

If that is the definition that we accept for the `or` helper, we can locally define another `or` helper, which overrides the "global" `or` helper, then use that helper in this template.